src/cindex.jl

```diff
@@ -221,7 +221,11 @@ endof(tl::TokenList) = length(tl)
 done(tl::TokenList, i) = (i > length(tl))
 
 function getindex(tl::TokenList, i::Int)
-    if (i < 1 || i > length(tl)) throw(BoundsError()) end
+    if (i < 1 || i > length(tl))
+        # throw(BoundsError())
+        warn("TokenList miss at index ", i)
+        return Comment(string("TokenList miss at index ", i))
```

Comment:
I found the problem. Will merge this plus some other fixes this weekend. Thanks!